src/library/traktor/traktorfeature.h

```diff
@@ -80,6 +82,7 @@ class TraktorFeature : public BaseExternalLibraryFeature {
     QString m_title;
 
     QSharedPointer<BaseTrackCache> m_trackSource;
+    WLibrarySidebar *m_pSidebarWidget;
```

Comment:
And here.

By the way, do all features have this member? If yes, it can become a protected member of the base class.

Reply:
Tracks, Analyze and Recording don't have it, at least they don't need it.
If thy do for feature right-click features we can move this to the base class.

Reply:
Ok, so the current solution is fine.